### ISSUE TITLE
MacOS: cannot unlock developer tab

### ISSUE THREAD

**Issue by rastagraffix** [mooltipass/moolticute#979]
### Expected behavior
The developer tab should appear.

### Actual behavior
The shift+fn+cmd+f3 dance is not unlocking the developer tab for me ...

### Step by step guide to reproduce the problem
Type the nearly untypeable keyboard sequence above. Nothing happens.

### Moolticute Version
0.53.7

### Operating System
MacOS 12.1 and 12.2. M1 CPU.
 
### Mooltipass Device
MiniBLE

**Comment by rastagraffix** [comment]
As I mentioned in #978, a simpler keyboard chord would be nice. CMD-Shift-D perhaps?

Also, Windows and Linux aren't options -- I don't run those on anything here. MacOS and OpenBSD are the choices.

**Comment by GuillaumeHemmen** [comment]
Hello @rastagraffix, can you try the same sequence but in this order:

- Press shift
- Press FN
- press CMD
- press F3

This is bit weird but I have to press the key in this exact sequence to have any "hidden" tab displayed.

**Comment by rastagraffix** [comment]
GuillaumeHemmen writes:

> This is bit weird but I have to press the key in this exact sequence to have 
> any "hidden" tab displayed.

No joy :-(

**Comment by limpkin** [comment]
@deXol what simpler shortcuts could we use?

**Comment by KDederichs** [comment]
my hide it behind a keyboard shortcut at all? Maybe displaying it by default (or at least having it in the application menu bar) would be much simpler (and people wouldn't have to look up the key combo all the time if they want to update their bundle)

**Comment by rastagraffix** [comment]
With the 0.55.0 moolticute build the Shift-Fn[1-3] keys now work on my MacBook M1.

**Comment by warrenc5** [comment]
> @deXol what simpler shortcuts could we use?

IMHO these trapdoor style hidden "Easter egg" features are all together a bad idea. - Even the right click on the icon is too obscure.   (At least put a tooltip on the icon saying right click to access hidden features). Maybe I'm showing my age but don't Apples only have one mouse button? :)

I think users should have full control over their devices, including upgrading firmware. - limiting "useful" features like this only delays user gratification.  

Users explore the functionality of the device through the UI - not the docs/forums. 

We must put the user at the center of all our decisions. Guide the user to avoid making dangerous mistakes. (e.g Clicking this button could irrecoverably brick your device and wipe your sim card, rather than saying "Make sure you read the docs before clicking this") 

Overly formal and obscure information leads to bad user outcomes. Just be direct, assume our user does have half a brain. It will save you and everyone time in the long run.
 
For example I only just found out about the ability to store ssh keys. This would have been a great help over the past 3 years had I known about this on the UI.

I known I've had a flat battery for ages but never knew about the "recondition battery" - I thought the software could/should be able to do the discharge/rest/recharge cycle but only discovered it after large amounts of reading the forums and some luck at finding the magic key combinations.

Btw - I still don't know what "knock detection" as the UI simply says enable/disable knock detection. It sounds like a cool feature, I have turned it on "high" - but I still don't know what it does. - don't worry - 

As much as I love this project and all the contributors :)
